setup.py

```diff
@@ -21,6 +21,17 @@
 
 exec(open("pyxem/release_info.py").read())  # grab version info
 
+# Projects with optional features for building the documentation and running
+# tests. From setuptools:
+# https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies
+extra_feature_requirements = {
+    "tests": [
+    	"pytest>=5.0", 
```

Comment:
Picky:y Have you run black? So that lines left-align.

Reply:
yeah, that's a good catch